setuptools/msvc.py

```diff
@@ -716,9 +718,9 @@ def UniversalCRTSdkDir(self):
         for ver in vers:
             sdkdir = self.ri.lookup(self.ri.windows_kits_roots, 'kitsroot%s' % ver)
             if sdkdir:
-                return sdkdir or ''
+                return sdkdir
 
-        return None
+        return ''
```

Comment:
This one I'm not entirely certain of, and does change the possible return types.

dir-related properties of this class are a mixed bag on whether they return `None` or `''` as a fallthrough case. And are almost never documented as returning None (which is fixed in this PR).

However all other properties that *can* return `None` are checked for truthiness when used, this one isn't. (hence it was causing issues for https://github.com/pypa/setuptools/pull/4744)

The redundant `or ''` makes me think this may have been intended to not return `None`. But the true intention is unclear to me.